src/extra/AutoUI/index.tsx

```diff
@@ -268,10 +268,12 @@ export const AutoUI = <T extends AutoUIBaseResource<T>>({
 
 	return (
 		<Flex
+			height="100%"
 			flexDirection={Array.isArray(data) ? 'column' : undefined}
```

Comment:
@JSReds do you remember the reason that we swapped the direction based on the type?
Given that there is only a single Spinner component inside it, I don't think that does what we expect.
A weird side-effect of it though is the jumpiness of the spinner in the Organizations page, right after the data arrive.
```suggestion
			flexDirection="column"
```

Reply:
@thgreasi this change come from this PR https://github.com/balena-io-modules/rendition/pull/1481/files and I think this was only a mistake made thinking about the collection vs entity. I hadn't noticed this, unfortunately, when I reviewed the PR. In any case, I agree with what you said above, I am quite convinced that this can be removed.

Reply:
@myarmolinsky can you do this change in a follow-up PR and give it a try?